v3/pkg/application/dialogs_linux.go

```diff
@@ -10,10 +10,15 @@ func (a *linuxApp) showAboutDialog(title string, message string, icon []byte) {
 	about.SetTitle(title).
 		SetMessage(message).
 		SetIcon(icon)
-	runQuestionDialog(
-		pointer(parent),
-		about,
-	)
+	InvokeAsync(func() {
+		response := runQuestionDialog(pointer(parent), about)
+		if response >= 0 && response < len(about.Buttons) {
+			button := about.Buttons[response]
+			if button.Callback != nil {
+				go button.Callback()
```

Comment:
The about show shouldn't be able to have specific handlers afaik.

At least it doesn't on any other platform.  Would recommend putting that logic back they way it was except for the asynchronous call change